mutt/dump-ical.py

```diff
@@ -4,6 +4,7 @@
 import warnings
 import vobject
 import datetime
+from tzlocal import get_localzone
```

Comment:
I take it `tzlocal` is a package you have installed locally?

Unfortunately that will mean this line adds a new python package dependency.

However, from a quick search it looks like Python has [builtin support for `astimezone`](https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone).

I'm also in favour taking a patch that will render the calendar invite in the user's timezone if we can do so with builtin library calls. If that means explicitly using Python 3 then that's fine, so long as we update the shebang at the top. Also would want to make sure that we render a suffix on the end of the timestamp so the user knew the timezone that is being used. I'd be happy with either the letter abbreviation (e.g. PST) or an offset (e.g. -08:00).

Reply:
> However, from a quick search it looks like Python has [builtin support for `astimezone`](https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone).

There is support in Python 3.9 (but not with the method you linked).

> I'm also in favour taking a patch that will render the calendar invite in the user's timezone if we can do so with builtin library calls. If that means explicitly using Python 3 then that's fine, so long as we update the shebang at the top. Also would want to make sure that we render a suffix on the end of the timestamp so the user knew the timezone that is being used. I'd be happy with either the letter abbreviation (e.g. PST) or an offset (e.g. -08:00).

I was not sure how render the event timezone as there is already the local timezone. Feel free to propose something different.